test/controllers/api/v2/provisioning_templates_controller_test.rb

```diff
@@ -96,6 +96,11 @@ class Api::V2::ProvisioningTemplatesControllerTest < ActionController::TestCase
     assert_response :unprocessable_entity
   end
 
+  test 'export should export the erb of the template' do
+    get :export, { :id => templates(:pxekickstart).to_param }
+    assert_response :success
```

Comment:
Some form of the content and/or response type (that it's an attachment with a given filename) should be tested here too.